src/mapValues.test-d.ts

```diff
@@ -82,6 +82,7 @@ describe("branded types", () => {
 
     mapValues(userValues, (value, key) => {
       expectTypeOf(value).toEqualTypeOf<number>();
+      // eslint-disable-next-line @typescript-eslint/no-unnecessary-template-expression -- TODO: I'm not sure what's going on here, are we doing something wrong or is this code needed so we can test it properly?
       expectTypeOf(key).toEqualTypeOf<`${UserID}`>();
```

Comment:
i remember branded types need to have a template expression for this to work properly